components/Banner/Banner.styled.ts

```diff
@@ -0,0 +1,30 @@
+import styled from 'styled-components';
+
+type MoneyProps = {
+  right?: boolean;
+};
+
+export const Background = styled.section`
+  position: fixed;
+  top: 64px;
+  width: 100%;
+  height: 40px;
+  background: #ebf4ee;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  color: #75b587;
+  font-size: 14px;
+  font-family: GilroySemiBold;
+  cursor: pointer;
+  z-index: 3;
+`;
+
+export const Money = styled.span<MoneyProps>`
+  font-size: 16px;
+  margin-right: ${({ right }) => (right ? '8px' : '0')};
+
+  &:last-of-type {
```

Comment:
In my previous comment, I mentioned that we should pass in a prop to conditionally margin-right / margin left. Let's use a prop instead of the `:last-of-type` pseudo-class to set up spacing to the right vs left of an emoji.